src/lib/sidebar/scheme_data.ts

```diff
@@ -39,20 +44,14 @@ export function backfill(json: SchemeCollection) {
     };
 
     // Move over the optional scheme_name from the old place
-    // @ts-ignore handling old format
```

Comment:
By declaring the input as `any`, we can get rid of most of this noise. `any` is more appropriate anyway; we don't expect the input to backfill to be valid at all!